game/templates/game/landing.html

```
@@ -134,7 +137,18 @@
             >CHECK<span class="logo-accent">ORA</span></span
           >
         </a>
-        <div class="nav-links">
+
+        <!-- HAMBURGER BUTTON -->
+    <button
+      class="nav-toggle"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)